src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/CryptographicAttributeObject.cs

```diff
@@ -29,6 +29,9 @@ public CryptographicAttributeObject(Oid oid, AsnEncodedDataCollection? values)
             {
                 foreach (AsnEncodedData asn in values)
                 {
+                    if (asn.Oid is null)
+                        throw new ArgumentException(SR.Argument_InvalidOidValue, nameof(values));
```

Comment:
Typically the braces are required unless you are at the "top" brace level of the method. We're not here, this is inside the `foreach` loop.

```suggestion
                    if (asn.Oid is null)
                    {
                        throw new ArgumentException(SR.Argument_InvalidOidValue, nameof(values));
                    }
```

I realize the `if` below is not setting a good example here. Would be nice to fix both.

Reply:
I fixed both the one I created and the if statement below it.